src/fal/packages/environments/base.py

```diff
@@ -40,7 +39,20 @@ def rmdir_on_fail(path: Path) -> Iterator[None]:
 
 
 def log_env(env: BaseEnvironment, message: str, *args, kind: str = "trace", **kwargs):
-    logger.log(kind.upper(), f"[{env.key}] {message}", *args, **kwargs)
+    message = f"[{env.key}] {message}"
+
+    if kind.startswith("trace"):
+        LOGGER.trace(message, *args, **kwargs)
+    elif kind.startswith("debug"):
+        LOGGER.debug(message, *args, **kwargs)
+    elif kind.startswith("info"):
+        LOGGER.info(message, *args, **kwargs)
+    elif kind.startswith("warn"):
+        LOGGER.warn(message, *args, **kwargs)
+    elif kind.startswith("error"):
+        LOGGER.error(message, *args, **kwargs)
+    else:
+        LOGGER.info(message, *args, **kwargs)
```

Comment:
```suggestion
    log_method = getattr(LOGGER, kind)
    log_method(message, *args, **kwargs)
```

Reply:
We lose the `startswith` check (think `warning` vs `warn`). Also, we would FAIL instead of defaulting to `info`. Maybe failing is better so the developer fixes the error...